### PR TITLE
Update README.md and license

### PR DESCRIPTION
**Pull request by konekoya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "node"
   ],
   "author": "Sam CH Cho",
-  "license": "ISC",
```
Comment (konekoya) [inline package.json:35]: The ISC license was probably generated by npm when initializing the project?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "node"
   ],
   "author": "Sam CH Cho",
```
Comment (konekoya) [inline package.json:34]: _CH_ seems like a typo for me

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ This is a wrapper encryption utility of [crypto-js](https://www.npmjs.com/packag
 - [Encrypt](#encrypt)
 - [Decrypt](#decrypt)
 
-# Quick Start
+## Quick Start
```
Comment (konekoya) [inline README.md:14]: Using h2 as the section headline is more commonly seen in the wild :)

File: package.json (hunk 1 above)
Comment (saivirtue) [reply package.json:35]: :)

File: package.json (hunk 2 above)
Comment (saivirtue) [reply package.json:34]: "CH" is the name abbreviation :)

**Pull request merged**